### PR TITLE
Update of repo from previous .net versions and include of last algorithms

### PR DESCRIPTION
**Pull request by bamartinezd**

### CODE REVIEW

File: MigratoryBirds/Program.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+using System.Collections.Generic;
+
+public class Program
+{
+    public static void Main(string []args){
+        var a = migratoryBirds(new List<int>{1,4,4,4,5,3});
+    }
+
+    public static int migratoryBirds(List<int> arr)
+    {
```
Comment (andrescollazosc) [inline MigratoryBirds/Program.cs:11]: why this function is return 0.

File: WordParserIntertec/Program.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1) The first letter of the word
+2) The number of distinct characters between first and last character
+3) The last letter of the word. 
+For example, Smooth would become S3h. 
+Words are separated by spaces or non-alphabetic characters and these separators should be maintained in their original form and location in the answer. 
+A few of the things we will be looking at is accuracy, efficiency, solution completeness. 
+*/
+public class Program
+{
+    public static string WordParser(string input)
```
Comment (andrescollazosc) [inline WordParserIntertec/Program.cs:30]: This method is too big, I think we could refactor it

File: TiendaPagoCodilityTest/Program.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        foreach (string element in arrElements)
+        {
+            int isNumeric;
+            if (int.TryParse(element, out isNumeric))
+            {
+                stack.Push(isNumeric);
+            }
+            else
+            {
+                switch (element)
```
Comment (andrescollazosc) [inline TiendaPagoCodilityTest/Program.cs:28]: This switch could be in another function.

File: MigratoryBirds/Program.cs (hunk 1 above)
Comment (bamartinezd) [reply MigratoryBirds/Program.cs:11]: Resolved, now it's returning expected value.

File: WordParserIntertec/Program.cs (hunk 2 above)
Comment (bamartinezd) [reply WordParserIntertec/Program.cs:30]: Right, can we refactor it together?

File: TiendaPagoCodilityTest/Program.cs (hunk 3 above)
Comment (bamartinezd) [reply TiendaPagoCodilityTest/Program.cs:28]: Right, can we refactor it together?

**Pull request closed without merging**